tron/core/actionrun.py

```diff
@@ -214,6 +214,8 @@ class ActionRun(Observable):
     EXIT_KUBERNETES_NOT_CONFIGURED = -7
     EXIT_KUBERNETES_TASK_INVALID = -8
     EXIT_KUBERNETES_ABNORMAL = -9
+    EXIT_KUBERNETES_SPOT_INTERRUPTION = 137
+    EXIT_KUBERNETES_NODE_SCALEDOWN = 143
```

Comment:
historically 137s indicated OOMs and 143s were other external kills e.g. via docker or via tronctl kill (see https://yelpwiki.yelpcorp.com/questions/111183238/answers/111183255)

I don't think we should override these expectations. 

It might make more sense to treat these special cases with negative exit codes as with the other system-level errors like KUBERNETES_NOT_CONFIGURED.

Reply:
The only reason why I used those numbers is so that it matches with the error codes from the events we were getting from Kubernetes. I can change it to negative error codes but then it wouldn't be the same code as the events we are getting from K8s.

Reply:
i'm not really sure why k8s has decided to co-opt `137` as an exit code for anything other than OOMKills :/

that said, i think it's perfectly fine to not have the tron exit code here match the k8s exit code (especially when the k8s exit code would be actively misleading) :)